admin_guide/configuring_authentication.adoc

```diff
@@ -570,17 +557,6 @@ is configured to trust the `X-Remote-User` header.
     Require valid-user
     RequestHeader set X-Remote-User %{REMOTE_USER}s
   </ProxyMatch>
-
-  # All other requests should use Bearer tokens.  These can only be verified by
```

Comment:
Might be worth redirecting all other traffic to the API, so `example.com/foo` redirects to `openshift.com:8443/foo`.

That would let you hit the auth proxy at `/` and get directed into the console login flow